content/api/editor-support.md

```diff
@@ -4,26 +4,26 @@ title: Editor support
 
 Installing our code locally will allow your editor to extract information from our code, giving you additional features like auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
 
-Installing these definitions is optional, however will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
+Installing these definitions is optional, but will be incredibly useful if your editor supports this. All editors in our [recommended list](/tutorials/editors/) besides IDLE will support it.
 
 ## Installation
-Installing our API, the code you interact with, can be installed using `pip`, Python's built-in package manager. `pip` should be installed on your machine, if it's not, you'll need to look up how to install it. See [here](https://stackoverflow.com/a/12476379) for troubleshooting on Windows.
+Our API, the code you interact with, can be installed using Python's built-in package manager `pip`. Assuming you have Python installed on your machine, `pip` should be installed as well. If it's not, you'll need to look up how to install it. See [here](https://stackoverflow.com/a/12476379) for troubleshooting on Windows.
 
 ```bash
 pip install git+https://github.com/sourcebots/robot-api
 ```
 
-If you're doing this on a shared computer, such as those you may find in your colleges, add `--user` to the end:
+The above command requires administrator access to the computer. If you're doing this on a shared computer, such as those you may find in your colleges, add `--user` to the end:
```

Comment:
`pip install` doesn't need administrator access (at least not in all configurations). This change also feels like it should have been a separate PR.

Reply:
It does when installing globally right? Modifying the system environment like that usually requires admin privileges

Reply:
I'm not sure. I've got it installed it via `chocolatey` so I'm not actually sure what options were set when it installed, though the Python directory is in the location which I expect when installing it "globally":
```
C:\Users\Peter>where pip
C:\Python36\Scripts\pip.exe
C:\Python27\Scripts\pip.exe
```

I think we want to say something like "Th above command _may_ require administrator.."

Reply:
I've got it installed using the installers on python.org. It installs to a different location to yours, howerver doesn't seem to require admin. On my linux machine however, I did need root or `--user`. I think perhaps moving this from inline in the instructions, and just adding a note about possibly needing `sudo`/`--user` in a notice or something would be the best solution. Certainly outside the scope of this PR

Reply:
I agree there's more that could be done; I think that adding a "may" to the current sentence improves it substantially (for all platforms) though.

Reply:
I've added a "may" as suggested (b4838f2aac7d6f0583b547adfb2e058a545dc085), but this could certainly do with some more investigation.

Reply:
Have addressed in https://github.com/sourcebots/docs/pull/79